dev/manual_tests/lib/density.dart

```diff
@@ -252,7 +252,8 @@ class _OptionsState extends State<Options> {
                           min: VisualDensity.minimumDensity,
                           max: VisualDensity.maximumDensity,
                           onChanged: (double value) {
-                            widget.model.density = widget.model.density.copyWith(horizontal: value, vertical: widget.model.density.vertical);
+                            widget.model.density = widget.model.density.copyWith(
+                                horizontal: value, vertical: widget.model.density.vertical);
```

Comment:
nit: the formatting seems off. I would suggest the following

```suggestion
                            widget.model.density = widget.model.density.copyWith(
                              horizontal: value,
                              vertical: widget.model.density.vertical,
                            );
                            
```

Reply:
Same below.

Reply:
Fixed, and below.